### PR TITLE
Macro declarations

### PR DESCRIPTION
**Pull request by DougGregor**

### CODE REVIEW

**Comment by DougGregor** [comment]
@swift-ci please test

**Comment by DougGregor** [comment]
@swift-ci please test

File: Sources/SwiftSyntaxBuilder/CMakeLists.txt~ [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This source file is part of the Swift.org open source project
```
Comment (bnbarham) [inline Sources/SwiftSyntaxBuilder/CMakeLists.txt~:1]: Looks like you accidentally added this and the one in `_SwiftSyntaxMacros`

**Comment by DougGregor** [comment]
@swift-ci please test

**Comment by DougGregor** [comment]
https://github.com/apple/swift/pull/62204

@swift-ci please test macOS

**Comment by DougGregor** [comment]
https://github.com/apple/swift/pull/62204

@swift-ci please test macOS

**Comment by DougGregor** [comment]
https://github.com/apple/swift/pull/62204

@swift-ci please test Linux

**Comment by DougGregor** [comment]
https://github.com/apple/swift/pull/62204

@swift-ci please test macOS

**Pull request merged**